src/components/Search.js

```diff
@@ -63,7 +63,7 @@ useEffect(()=>
     trackPromise(
     axios.get(url)
         .then(res=>{
-            console.log(res.data);
+            console.log(res.data, "first time fetch");

```

Comment:
Remove this "first time fetch" from console.log

Reply:
Fixed everything